Sources/WalletConnectPush/Client/Wallet/ProtocolEngine/wc_notifyUpdate/NotifyUpdateResponseSubscriber.swift

```diff
@@ -50,7 +50,7 @@ class NotifyUpdateResponseSubscriber {
                     
                     let updatedSubscription = PushSubscription(topic: subscriptionTopic, account: oldSubscription.account, relay: oldSubscription.relay, metadata: oldSubscription.metadata, scope: scope, expiry: expiry)
 
-                    subscriptionsStore.set(updatedSubscription, forKey: subscriptionTopic)
+                    try await subscriptionsStore.set(object: updatedSubscription, for: updatedSubscription.account)
```

Comment:
why do we get for topic and set for account?

Reply:
account is using to get syncTopic (SyncStore line 68)